frontend/views/pages/PendingApproval.vue

```diff
@@ -36,7 +37,7 @@ export default ({
     ...mapState(['currentGroupId']),
     ourGroupProfile () {
       if (!this.ephemeral.groupIdWhenMounted) return
-      return this.$store.state[this.ephemeral.groupIdWhenMounted]?.profiles?.[this.ourUsername]
+      return this.$store.state[this.ephemeral.groupIdWhenMounted]?.profiles?.[this.ourUsername]?.status === PROFILE_STATUS.ACTIVE
```

Comment:
This change changes the meaning of this computed property.

The computed property is named `ourGroupProfile`, which implies it returns our group profile. But now this no longer happens, instead it returns a boolean.

So either the condition needs to be removed or the name of the computed property needs to be changed.

Reply:
Agreed.